clients/ptranslator/src/main/java/org/hawkular/metrics/clients/ptrans/PTrans.java

```diff
@@ -60,7 +60,7 @@
  * @author Thomas Segismont
  */
 public class PTrans {
-    private static final Logger LOG = LoggerFactory.getLogger(PTrans.class);
+    private static final PTransLogger log = PTransLogging.getPTransLogger(PTrans.class);
```

Comment:
I don't understand. How is it different from, for example, `RestLogger`, or `CoreLogger` ?

See https://github.com/hawkular/hawkular-metrics/pull/352/files#diff-9c93f73c3ccf924ff0d6ef4edf4219feR89